### DISCUSSION TITLE
Published an article on using Kestrel/ASP.NET for creating and accessing gRPC services

### DISCUSSION THREAD

**Discussion by npolyak** [grpc/grpc-web#1423] [Show and tell]
Published an article on using Kestrel/ASP.NET for creating and accessing gRPC services: [gRPC, .NET 8.0 and Kestrel in Easy Samples](https://www.codeproject.com/Articles/5382064/gRPC-NET-8-0-and-Kestrel-in-Easy-Sample)
The article covers various gRPC paradigms including single-response/single-reply, server streaming, client streaming and bi-directional streaming.
Also 3 types of clients are demonstrated - C# console client, HTML/JavaScript Web Browser client and C# non-Blazor WebAssembly Avalonia Web Browser client. The last two clients are using are using grpc-web.

**Comment by sampajano** [comment]
@npolyak Wonderful! Thank you for your mention and sharing it here! 😃

**Reply by npolyak** [reply]
Pleasure :)
